doc/develop/tools/stm32cubeide.rst

```diff
@@ -0,0 +1,112 @@
+.. _stm32cube_ide:
+
+STM32CubeIDE
+############
+
+STM32CubeIDE_ is designed for the STM32 series of chips from STMicroelectronics.
+
+This guide describes the process of setting up, building, and debugging Zephyr
+applications using the IDE.  Using the STM32CubeIDE allows you to build and
+debug Zephyr applications as if they were created in the IDE using ST's
+toolset.
+
+A project must have already been created with Zephyr and west.
+
+These directions have been validated to work with 1.16.0 of the IDE.
+
+These directions have been adapted from a `GitHub discussion <https://github.com/zephyrproject-rtos/zephyr/discussions/69812#discussioncomment-8770986>`_.
+
+Project Setup
+*************
+
+#. Setup a Zephyr workspace such that west is available.  This can be followed
+   in the `Getting Started Guide <https://docs.zephyrproject.org/latest/getting_started/index.html>`_.
+   You will need to install the SDK and make sure it is available.
+
+#. Activate your environment.
+
+#. Run STM32CubeIDE from your Zephyr environment.  Example:
+
+   .. code-block::
+
+      $ /opt/st/stm32cubeide_1.16.0/stm32cubeide
+
+#. Open up your already existing project by going to
+   :menuselection:`File->New->STM32 CMake Project`:
+
+   .. image:: img/stm32cube_new_cmake.png
+      :align: center
```

Comment:
please make the `image::`s `figure::`s instead, including a caption and an alt description (see https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#images). This will add some breathing room in the text but also help with accessibility